test/parallel/test-timers-unrefed-in-callback.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+// Checks that setInterval timers keep running even when they're
+// unrefed within their callback.
+
+require('../common');
+const assert = require('assert');
+const net = require('net');
+
+let counter1 = 0;
+let counter2 = 0;
+
+// Test1 checks clearInterval also works fine for the unrefed timer
+// within timer callback. counter1 stops at exactly 11 with
+// clearInterval. This test should be executed at first to confirm
```

Comment:
@misterdjules It describes much more details for Test1. There is no reason against for it. Fixed in dffeb3b97915ec3582205dfa883cac0aaf1f9ad4.